Assets/Scripts/Player/PlayerControls.cs

```diff
@@ -229,7 +285,15 @@ public override void Update()
 				}
 				else
 				{
-					player.physics.ChangeState( PhysicsStateType.Grounded );
+					if ( player.controls.jumpButton.down &&
```

Comment:
I've been trying to give the player as much opportunity to jump as possible, which includes allowing the player to jump again before they've quite hit the ground. I figure this is in the vein of the late jump delay. That said, I reworked the logic of this one to make it a bit clearer.